x-pack/plugins/security_solution/public/detections/components/endpoint_responder/use_responder_action_item.tsx

```diff
@@ -32,15 +33,20 @@ export const useResponderActionItem = (
     [eventDetailsData]
   );
 
-  const agentType: ResponseActionAgentType = useMemo(
-    () =>
-      eventDetailsData
-        ? isAlertFromSentinelOneEvent({ data: eventDetailsData })
-          ? 'sentinel_one'
-          : 'endpoint'
-        : 'endpoint',
-    [eventDetailsData]
-  );
+  const agentType: ResponseActionAgentType = useMemo(() => {
+    if (!eventDetailsData) {
+      return 'endpoint';
+    }
+
+    if (isAlertFromSentinelOneEvent({ data: eventDetailsData })) {
+      return 'sentinel_one';
+    }
+    if (isAlertFromCrowdstrikeEvent({ data: eventDetailsData })) {
```

Comment:
We needed this check here to not default to `endpoint` when the alert detail is for a crowdstrike alert.